src/api/layers/WMSCapabilitiesParser.class.js

```diff
@@ -171,15 +248,18 @@ export default class WMSCapabilitiesParser {
         return layerExtent
     }
 
-    getLayerAttribution(layer) {
+    _getLayerAttribution(layer) {
         let title = null
         let url = null
         if (layer.Attribution || this.Capability.Layer.Attribution) {
             const attribution = layer.Attribution || this.Capability.Layer.Attribution
             url = attribution.OnlineResource
-            title = attribution.Title || new URL(attribution.OnlineResource).hostname
+            title =
+                attribution.Title || new URL(attribution.OnlineResource || this.originUrl).hostname
```

Comment:
`new URL` may throw a `TypeError` if the given URL is malformed/invalid, it might be a good idea to drag the `ignoreError` from the parent call here, and deal with any error with a `try...catch` and decide depending on `ignoreError` what to do with them

Reply:
or wrap the call to this function in a `try...catch` in the parent, if that make more sense business wise

Reply:
Done